src/applications/vaos/services/healthcare-service/transformers.js

```diff
@@ -15,121 +15,14 @@
  */
 export function transformAvailableClinic(facilityId, typeOfCareId, clinic) {
   return {
+    // ID: VA facility site code and clinic id
     id: `${clinic.siteCode}_${clinic.clinicId}`,
-    resourceType: 'HealthcareService',
-    // External identifiers for this item
-    identifier: [
-      {
-        system: 'http://med.va.gov/fhir/urn',
-        value: `urn:va:healthcareservice:${clinic.siteCode}:${facilityId}:${
-          clinic.clinicId
-        }`,
-      },
-    ],
-
-    // Organization that provides this service
-    providedBy: `Organization/${clinic.siteCode}`,
-
-    // Specific service delivered or performed
-    serviceType: [
-      {
-        // Type of service delivered or performed.
-        type: {
-          // Code defined by a terminology system
-          coding: {
-            // A symbol in syntax defined by the system.
-            code: typeOfCareId,
-            userSelected: false,
-          },
-        },
-      },
-    ],
-
-    // Location where service may be provided
-    location: {
-      reference: `Location/${facilityId}`,
-    },
-
+    // ID of physical VA Facility (sta6id) where the clinic is located
+    stationId: facilityId,
+    // Name of VA facility where clinic is located
+    stationName: clinic.institutionName,
     // Description of service as presented to a consumer while searching
-    serviceName: clinic.clinicFriendlyLocationName
-      ? clinic.clinicFriendlyLocationName
-      : clinic.clinicName,
-
-    // Collection of characteristics (attributes)
-    // NOTE: The following pattern is used to populate this section (code & display are mutally exclusive):
-    // 1. The 'text' attribute is used as the name of the code.
-    // 2. The 'code' attribute is used if the attribute is (or looks like a code)
-    // else it's undefined
-    // 2. The 'display' attribute is used to hold the actual value of the code.
-    characteristic: [
```

Comment:
`institutionName`, `institutionCode`, and `clinicFriendlyLocationName` now map to the following: `stationName`, `stationId`, and `serviceName`

Could not find any necessary references to `directSchedulingFlag` and `displayToPatientFlag`.

Reply:
We only get clinics that have those flags set to true, generally, so I don't believe we ever have a need to check them.